docs/generated-code-usage.md

```diff
@@ -30,9 +30,8 @@ somewhere in your code:
     System.loadLibrary("YourLibraryName");
     // The name is specified in Android.mk / build.gradle / Makefile, depending on your build system.
 
-If you package your native library in a jar, you can also use `com.dropbox.djinni.NativeLibLoader` 
-to help unpack and load your lib(s).  See the [Localhost README](example/localhost/README.md)
-for details.
+If you package your native library in a jar, you can also use the [`NativeLibLoader`](https://github.com/cross-language-cpp/djinni-support-lib/blob/main/java/com/dropbox/djinni/NativeLibLoader.java)
```

Comment:
I opened [an issue about](https://github.com/cross-language-cpp/djinni-support-lib/issues/8) the `NativeLibLoader`. Not sure what this is about.

Reply:
We have our own lib loader at work, so I also don't know about that one. But it's good to look at it